tests/EngineTest.php

```diff
@@ -23,41 +24,42 @@ public function testCanCreateInstance()
     public function testSetDirectory()
     {
         $this->assertInstanceOf('League\Plates\Engine', $this->engine->setDirectory(vfsStream::url('templates')));
-        $this->assertEquals($this->engine->getDirectory(), vfsStream::url('templates'));
+        $this->assertSame(vfsStream::url('templates'), $this->engine->getDirectory());
     }
 
     public function testSetNullDirectory()
     {
         $this->assertInstanceOf('League\Plates\Engine', $this->engine->setDirectory(null));
-        $this->assertEquals($this->engine->getDirectory(), null);
+        $this->assertNull($this->engine->getDirectory());
     }
 
     public function testSetInvalidDirectory()
     {
-        $this->setExpectedException('LogicException', 'The specified path "vfs://does/not/exist" does not exist.');
+        // The specified path "vfs://does/not/exist" does not exist.
+        $this->expectException(\LogicException::class);
```

Comment:
you can use `$this->expectExceptionMessage('...')` here, to ensure that the message is the same as the one expected.